test-cases/longevity/longevity-twcs-3h.yaml

```diff
@@ -21,3 +21,5 @@ nemesis_during_prepare: false
 space_node_threshold: 64424
 
 user_prefix: 'longevity-twcs-3h'
+
+post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 300;"
```

Comment:
Here we don't have TTL and timewindow size?

Reply:
as we discussed, the default values are ok for 3h
timewindow size 1 minute. ttl = 900

Reply:
ok.